.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```

Comment:
Comments are not allowed in json. Also, are these supposed to be here? LOL

Reply:
Yes, this has always been here in VSC generated template files. I guess they use `json-c` which IMO should just take over `json` at this point.